### PR TITLE
Update payment page

### PR DESCRIPTION
**Pull request by gmrz**
Update payment page to pay per user (total items), also divide extra pays into persons and not into items.
Update detail page

### CODE REVIEW

File: src/main/java/org/jala/efeeder/details/DetailsCommand.java [hunk 1]
```diff
@@ -216,4 +219,17 @@ private Timestamp getPaymentTime(Connection connection, String idFoodMeeting) th
...
 		return foodMeetingManager.getFoodMeetingById(Integer.parseInt(idFoodMeeting)).getPaymentDate();
 	}
+	
+	private List<User> getUsersWithOrders(Connection connection, int idFoodMeeting) throws SQLException {
+		UserManager userManager = new UserManager(connection);
+		return userManager.getUsersWithOrders(idFoodMeeting);
+	}
+	
+	private double getSharedPriceByUser(double itemTotalPrice, int usersSize) {
+		double result = 0.0;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/details/DetailsCommand.java]: this value is never used

File: src/main/java/org/jala/efeeder/details/DetailsCommand.java [hunk 2]
```diff
@@ -216,4 +219,17 @@ private Timestamp getPaymentTime(Connection connection, String idFoodMeeting) th
...
+		
+		result = itemTotalPrice > 0 ? (usersSize > 0 ? itemTotalPrice / usersSize : 0) : 0;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/details/DetailsCommand.java]: there are two conditional in one single line, please use 'if' statement in order to get the code more readable

File: src/main/java/org/jala/efeeder/payment/PaymentCommand.java [hunk 3]
```diff
@@ -98,23 +98,16 @@ private double getTotalExternalItemPrice(List<PaymentItem> itemList) {
...
-		return orderManager.getOrdersWithUserByFoodMeeting(Integer.parseInt(idFoodMeeting));
+	private List<User> getUsersWithOrders(Connection connection, String idFoodMeeting) throws SQLException {
+		UserManager userManager = new UserManager(connection);
+		return userManager.getUsersWithOrders(Integer.parseInt(idFoodMeeting));
 	}
 
-	private Order extractMyOrder(int idUser, List<Order> orders) {
-		Order myOrder = null;
+	private double getSharedPriceByUser(double itemTotalPrice, int usersSize) {
+		double result = 0.0;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/payment/PaymentCommand.java]: value is never used

File: src/main/java/org/jala/efeeder/payment/PaymentCommand.java [hunk 4]
```diff
@@ -98,23 +98,16 @@ private double getTotalExternalItemPrice(List<PaymentItem> itemList) {
...
+		
+		result = itemTotalPrice > 0 ? (usersSize > 0 ? itemTotalPrice / usersSize : 0) : 0;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/payment/PaymentCommand.java]: use if statement

File: src/main/java/org/jala/efeeder/details/DetailsCommand.java (hunk 1 above)
Comment (gmrz) [reply src/main/java/org/jala/efeeder/details/DetailsCommand.java]: done!

File: src/main/java/org/jala/efeeder/details/DetailsCommand.java (hunk 2 above)
Comment (gmrz) [reply src/main/java/org/jala/efeeder/details/DetailsCommand.java]: done!

File: src/main/java/org/jala/efeeder/payment/PaymentCommand.java (hunk 3 above)
Comment (gmrz) [reply src/main/java/org/jala/efeeder/payment/PaymentCommand.java]: done!

File: src/main/java/org/jala/efeeder/payment/PaymentCommand.java (hunk 4 above)
Comment (gmrz) [reply src/main/java/org/jala/efeeder/payment/PaymentCommand.java]: done!

**Comment by gmrz** [comment]
Code was updated based on CR's comments

**Review by Minbozz — APPROVED** [review]

**Pull request merged**